bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/obscomp/SPSiteQuality.java

```diff
@@ -197,8 +199,18 @@ public SPSiteQuality clone() {
 
         PERCENT_20("20%/Darkest", 20, 21.37),
         PERCENT_50("50%/Dark", 50, 20.78),
-        PERCENT_80("80%/Grey", 80, 19.61) { public Magnitude adjust(Magnitude mag) { return adjustIf(R, mag, -0.3); } },
-        ANY("Any/Bright", 100, 0)         { public Magnitude adjust(Magnitude mag) { return adjustIf(R, mag, -0.5); }
+        PERCENT_80("80%/Grey", 80, 19.61) {
+            private double adjustment = -0.3;
+            @Override
+            public Magnitude adjust(Magnitude mag) {
+                return adjustIf(r, adjustIf(R, adjustIf(UC, mag, adjustment), adjustment), adjustment);
```

Comment:
I suppose we could factor out this multi-band `adjustIf` since we need it five times.